### ISSUE TITLE
feat(next): keyboard control accessibility

### ISSUE THREAD

**Issue by Blakintosh** [Blakintosh/svelte-flexiboards#23]
This is a high priority issue as it's important for accessibility. The user should be able to manipulate (grab and resize) widgets in a board and across targets using the keyboard only. 

The premise would be: Tab/Shift+Tab to tab through widgets (as already exists), then Enter on grabber/widget itself to grab a widget, Arrow Keys to move the widget around once grabbed, Enter again to drop, or Esc to cancel move. Then for resize, Enter to start resize, Arrow Keys to adjust size, Enter to confirm, Esc to cancel.

Should also decide whether to use a focus trap, and use an announcer for grab/resize events for screenreaders.
